simulationTool/components/SectionHeader.vue

```diff
@@ -9,7 +9,12 @@ export default {
       icon: {
         type: String,
         required: false
-      }
+      },
+      isVisible: { 
+        type: Boolean,
+        required: false,
+        default: true
+    }
```

Comment:
Is this used?

Reply:
Good pick! This is a leftover from an older version .